src/utils/claim.ts

```diff
@@ -325,8 +330,17 @@ export const claim = async <T extends ReverseSwap | ChainSwap>(
     }
 
     log.debug("Broadcasting claim transaction");
+
+    if (externalBroadcast) {
+        const res = await broadcastToExplorer(asset, claimTransaction.toHex());
+        log.debug("Claim transaction broadcast result via explorer", res);
+        if (res.id) {
+            swap.claimTx = res.id;
+        }
+    }
+
     const res = await broadcastTransaction(asset, claimTransaction.toHex());
-    log.debug("Claim transaction broadcast result", res);
+    log.debug("Claim transaction broadcast result via backend", res);
```

Comment:
Let's abstract this logic, because it's the exact same for refund broadcasts. Also, let's broadcast to both in parallel and use `Promise.allSettled`; as long as one of the two endpoints works, we are fine